sandbox/src/mocks/getCodeSystem/responses/AppointmentCancellationReasonCodeSystem.json

```diff
@@ -6,10 +6,10 @@
         ]
     },
     "resourceType": "CodeSystem",
-    "url": "https://api.dev3.ers.ncrs.nhs.uk/ers-api/STU3/v1/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
+    "url": "https://ers-api:445/STU3/v1/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
```

Comment:
I think this should start with '_baseUrl_'

Reply:
some more xamples of thus further down, its its best to just search for https://ers-api:445 and replace with _baseUrl_